.rubocop_todo.yml

```diff
@@ -1,6 +1,6 @@
 # This configuration was generated by
-# `rubocop --auto-gen-config --auto-gen-only-exclude`
```

Comment:
Previously this was generated using `--auto-gen-only-exclude` which I believe is the preferred way because it shows you which files need work.

Reply:
re-generated with that switch

Reply:
wow - that's cool to know.